### DISCUSSION TITLE
How to assign users to orgs before they login when configuring grafana with keycloak?

### DISCUSSION THREAD

**Discussion by ahmedTouati** [grafana/grafana#44787] [General]
We have configured grafana with keycloak, and we want to assign users to orgs before they login but it seems that users are only accessible in grafana when they login. Is there any solution to that?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
